packages/cxl-ui/package.json

```diff
@@ -23,7 +23,11 @@
     "@vaadin/vaadin-themable-mixin": "^1.5.2",
     "headroom.js": "^0.11.0",
     "lit-element": "^2.2.1",
-    "lit-html": "^1.1.2"
+    "lit-html": "^1.1.2",
+    "cross-env": "~7.0.2",
+    "laravel-mix": "^5.0.4",
+    "postcss-preset-env": "^6.7.0",
```

Comment:
@saas786 why do we need `postcss-preset-env` here, since we have our own SASS stack already?